ferrilata-devilline.IntegrationTests/Scenarios/IntegrationTests.cs

```diff
@@ -12,5 +21,185 @@ public IntegrationTests(TestContext testContext)
         {
             _testContext = testContext;
         }
+
+        [Theory]
+        [InlineData("api/post/pitch")]
+        public async Task PostPitchCorrect_AuthorizationPresent(string url)
+        {
+            var newPosting = new Pitch
+            {
+                BadgeName = "English speaker",
+                OldLVL = 2,
+                PitchedLVL = 3,
+                PitchMessage = "Hello World! My English is bloody gorgeous.",
+                Holders = new[] { "balazs.jozsef", "benedek.vamosi", "balazs.barna" }.ToList()
+            };
+
+            var PostingJson = JsonConvert.SerializeObject(newPosting);
```

Comment:
SerializeObject would return a string, which should be stored in a string, not in a var

Reply:
You could use simple strings without conversions as a test's expected value, but since it's a complex object, it could change later, so it's okey

Reply:
checked, thanks